packages/mainsail/source/address.service.ts

```diff
@@ -75,7 +75,7 @@ export class AddressService extends Services.AbstractAddressService {
 		privateKey: string,
 		options?: Services.IdentityOptions,
 	): Promise<Services.AddressDataTransferObject> {
-		const keyPair = await this.#keyPairFactory.fromPrivateKey(Buffer.from(privateKey));
+		const keyPair = await this.#keyPairFactory.fromPrivateKey(Buffer.from(privateKey, "hex"));
```

Comment:
Fixed a bug, should be in `hex` format

https://helloardent.slack.com/archives/C07QTEQJK5K/p1734009379022569